tests/unit_test/data/jobs/duplicate_clients/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "sag",
+  "study_name": "",
+  "resource_spec": {
+    "site-1": {
+      "gpu": 1
+    },
+    "site-2": {
+      "gpu": 1
+    }
+  },
+  "deploy_map": {
+    "sag1": [
+      "server",
+      "site-a",
+      "site-b"
```

Comment:
is it intentional that "resource_spec" has "site-1" and "site-2" while "deploy_map" has "site-a" and "site-b"?